pom.xml

```diff
@@ -536,7 +536,7 @@
                 <plugin>
                     <groupId>org.owasp</groupId>
                     <artifactId>dependency-check-maven</artifactId>
-                    <version>3.3.0</version>
+                    <version>3.3.1</version>
```

Comment:
bumped this version to avoid https://github.com/jeremylong/DependencyCheck/issues/1394